packages/core/src/index.ts

```diff
@@ -2,19 +2,48 @@ export * from './connectivity';
 export * from './header-util';
 export * from './http-client';
 export * from './odata-common';
-export { entityDeserializer } from './odata-common';
+// The explicit export is needed to guarantee backwards compatibility, it overrides the same named function in odata-v4.
```

Comment:
Maybe you can put the two comments together as one and phrase it something like: "... exports are needed ..." and "named exports in the other odata modules"

Reply:
Alright, I've updated the comment and restructured the explicit exports to be adjacent to each other